phpBB/includes/mcp/mcp_ban.php

```diff
@@ -28,7 +28,10 @@ function main($id, $mode)
 		global $db, $user, $auth, $template, $request, $phpbb_dispatcher;
 		global $phpbb_root_path, $phpEx;
 
-		include($phpbb_root_path . 'includes/functions_user.' . $phpEx);
+		if (!function_exists('user_ban'))
+		{
+			include($phpbb_root_path . 'includes/functions_user.' . $phpEx);
+		}
 
 		// Include the admin banning interface...
 		include($phpbb_root_path . 'includes/acp/acp_ban.' . $phpEx);
```

Comment:
Usually no, as admin modules shouldn't be included manually (unless in this exact case where we have some annoying legacy code).